shiny/Data-Ingest/server_files/input_record_svr.R

```diff
@@ -0,0 +1,40 @@
+inputsList <- list()
+bety <- betyConnect()
+
+######### Select Site ###############
+  sites <- dplyr::tbl(bety, "sites") %>% 
+    dplyr::select(one_of("sitename", "id")) %>% collect()
+  sitenames <- sites$sitename ## Is this a redundant step?
+  updateSelectizeInput(session, "InputSiteID", choices = sort(unique(sitenames)))
```

Comment:
FYI, this can be simplified (and sped up, though not a ton) as:

```r
sitenames <- dplyr::tbl(bety, "sites") %>% dplyr::distinct(sitename) %>% dplyr::arrange(sitename) %>% pull(sitename)
updateSelectizeInput(session, "InputSiteID", choices = sitenames)
```

Reply:
Alright so this is pushing 1:45 of loading time with the changes... Not sure this is going in the right direction XD